Its.Log.Monitoring/Telemetry.cs

```diff
@@ -0,0 +1,53 @@
+using System.Collections.Generic;
+using Its.Log.Instrumentation;
+
+namespace Its.Log.Monitoring
+{
+    public class Telemetry
+    {
```

Comment:
honestly we can use the same type here if we want a dependency between the two. has a few pros and cons. easier to keep them in sync that way... but then again you also then have to keep them in sync.